woocommerce/woocommerce-ios
# Update "sort and filter" product toolbar behaviour

Fixes #4516 
# Description
When a store doesn't have products, the app should hide the sort and filter buttons. 
If the user adds a product as a first one, the app should show again the buttons.

# Testing
Case 1: Store without products.
- Login with a store without products.
- Check that buttons don't appear.

Case 2: Add first product.
- Login with a store without products.
- Check that buttons don't appear.
- Add one product.
- Check that buttons appear.

Case 3: Remove last product.
- Get a Store with only one product.
- Remove this product.
- Check that buttons don't appear.

# Videos
start with one product
https://user-images.githubusercontent.com/11411847/124625749-5ea43f00-de7e-11eb-93ed-0100542ca815.mp4

start with 0 products
https://user-images.githubusercontent.com/11411847/124633692-d4f86f80-de85-11eb-84c8-530908cf7e0b.mp4

delete last product
https://user-images.githubusercontent.com/11411847/124625886-7d0a3a80-de7e-11eb-993b-700705d0c9b5.mp4

add first product
https://user-images.githubusercontent.com/11411847/124625970-8eebdd80-de7e-11eb-9a95-5de756e58fb6.mp4
